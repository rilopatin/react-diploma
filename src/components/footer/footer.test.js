import React from 'react'
import Footer from './footer'
import {shallow} from 'enzyme'


describe('Testing <Footer/>', () => {
    it('Footer has been rendered correctly', () => {
        const footer = shallow(<Footer/>)
        expect(footer).toMatchSnapshot();
    })
})