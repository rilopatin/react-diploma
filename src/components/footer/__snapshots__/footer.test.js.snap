// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Footer/> Footer has been rendered correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-lg-5 offset-lg-4"
          >
            <ul
              className="footer"
            >
              <li
                className="footer__item"
              >
                <Link
                  replace={false}
                  to="/"
                >
                  <img
                    alt="logo"
                    src="Logo_black.svg"
                  />
                </Link>
              </li>
              <li
                className="footer__item"
              >
                <Link
                  replace={false}
                  to="/coffee/"
                >
                  Our coffee
                </Link>
              </li>
              <li
                className="footer__item"
              >
                <Link
                  replace={false}
                  to="/goods/"
                >
                  For your pleasure
                </Link>
              </li>
            </ul>
          </div>
        </div>
        <img
          alt="Beans logo"
          className="beanslogo"
          src="Beans_logo_dark.svg"
        />
      </div>,
      "className": "footer-wrap",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="row"
          >
            <div
              className="col-lg-5 offset-lg-4"
            >
              <ul
                className="footer"
              >
                <li
                  className="footer__item"
                >
                  <Link
                    replace={false}
                    to="/"
                  >
                    <img
                      alt="logo"
                      src="Logo_black.svg"
                    />
                  </Link>
                </li>
                <li
                  className="footer__item"
                >
                  <Link
                    replace={false}
                    to="/coffee/"
                  >
                    Our coffee
                  </Link>
                </li>
                <li
                  className="footer__item"
                >
                  <Link
                    replace={false}
                    to="/goods/"
                  >
                    For your pleasure
                  </Link>
                </li>
              </ul>
            </div>
          </div>,
          <img
            alt="Beans logo"
            className="beanslogo"
            src="Beans_logo_dark.svg"
          />,
        ],
        "className": "container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="col-lg-5 offset-lg-4"
            >
              <ul
                className="footer"
              >
                <li
                  className="footer__item"
                >
                  <Link
                    replace={false}
                    to="/"
                  >
                    <img
                      alt="logo"
                      src="Logo_black.svg"
                    />
                  </Link>
                </li>
                <li
                  className="footer__item"
                >
                  <Link
                    replace={false}
                    to="/coffee/"
                  >
                    Our coffee
                  </Link>
                </li>
                <li
                  className="footer__item"
                >
                  <Link
                    replace={false}
                    to="/goods/"
                  >
                    For your pleasure
                  </Link>
                </li>
              </ul>
            </div>,
            "className": "row",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <ul
                className="footer"
              >
                <li
                  className="footer__item"
                >
                  <Link
                    replace={false}
                    to="/"
                  >
                    <img
                      alt="logo"
                      src="Logo_black.svg"
                    />
                  </Link>
                </li>
                <li
                  className="footer__item"
                >
                  <Link
                    replace={false}
                    to="/coffee/"
                  >
                    Our coffee
                  </Link>
                </li>
                <li
                  className="footer__item"
                >
                  <Link
                    replace={false}
                    to="/goods/"
                  >
                    For your pleasure
                  </Link>
                </li>
              </ul>,
              "className": "col-lg-5 offset-lg-4",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li
                    className="footer__item"
                  >
                    <Link
                      replace={false}
                      to="/"
                    >
                      <img
                        alt="logo"
                        src="Logo_black.svg"
                      />
                    </Link>
                  </li>,
                  <li
                    className="footer__item"
                  >
                    <Link
                      replace={false}
                      to="/coffee/"
                    >
                      Our coffee
                    </Link>
                  </li>,
                  <li
                    className="footer__item"
                  >
                    <Link
                      replace={false}
                      to="/goods/"
                    >
                      For your pleasure
                    </Link>
                  </li>,
                ],
                "className": "footer",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <Link
                      replace={false}
                      to="/"
                    >
                      <img
                        alt="logo"
                        src="Logo_black.svg"
                      />
                    </Link>,
                    "className": "footer__item",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": <img
                        alt="logo"
                        src="Logo_black.svg"
                      />,
                      "replace": false,
                      "to": "/",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "alt": "logo",
                        "src": "Logo_black.svg",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "img",
                    },
                    "type": [Function],
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <Link
                      replace={false}
                      to="/coffee/"
                    >
                      Our coffee
                    </Link>,
                    "className": "footer__item",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Our coffee",
                      "replace": false,
                      "to": "/coffee/",
                    },
                    "ref": null,
                    "rendered": "Our coffee",
                    "type": [Function],
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <Link
                      replace={false}
                      to="/goods/"
                    >
                      For your pleasure
                    </Link>,
                    "className": "footer__item",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "For your pleasure",
                      "replace": false,
                      "to": "/goods/",
                    },
                    "ref": null,
                    "rendered": "For your pleasure",
                    "type": [Function],
                  },
                  "type": "li",
                },
              ],
              "type": "ul",
            },
            "type": "div",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Beans logo",
            "className": "beanslogo",
            "src": "Beans_logo_dark.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col-lg-5 offset-lg-4"
            >
              <ul
                className="footer"
              >
                <li
                  className="footer__item"
                >
                  <Link
                    replace={false}
                    to="/"
                  >
                    <img
                      alt="logo"
                      src="Logo_black.svg"
                    />
                  </Link>
                </li>
                <li
                  className="footer__item"
                >
                  <Link
                    replace={false}
                    to="/coffee/"
                  >
                    Our coffee
                  </Link>
                </li>
                <li
                  className="footer__item"
                >
                  <Link
                    replace={false}
                    to="/goods/"
                  >
                    For your pleasure
                  </Link>
                </li>
              </ul>
            </div>
          </div>
          <img
            alt="Beans logo"
            className="beanslogo"
            src="Beans_logo_dark.svg"
          />
        </div>,
        "className": "footer-wrap",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="row"
            >
              <div
                className="col-lg-5 offset-lg-4"
              >
                <ul
                  className="footer"
                >
                  <li
                    className="footer__item"
                  >
                    <Link
                      replace={false}
                      to="/"
                    >
                      <img
                        alt="logo"
                        src="Logo_black.svg"
                      />
                    </Link>
                  </li>
                  <li
                    className="footer__item"
                  >
                    <Link
                      replace={false}
                      to="/coffee/"
                    >
                      Our coffee
                    </Link>
                  </li>
                  <li
                    className="footer__item"
                  >
                    <Link
                      replace={false}
                      to="/goods/"
                    >
                      For your pleasure
                    </Link>
                  </li>
                </ul>
              </div>
            </div>,
            <img
              alt="Beans logo"
              className="beanslogo"
              src="Beans_logo_dark.svg"
            />,
          ],
          "className": "container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="col-lg-5 offset-lg-4"
              >
                <ul
                  className="footer"
                >
                  <li
                    className="footer__item"
                  >
                    <Link
                      replace={false}
                      to="/"
                    >
                      <img
                        alt="logo"
                        src="Logo_black.svg"
                      />
                    </Link>
                  </li>
                  <li
                    className="footer__item"
                  >
                    <Link
                      replace={false}
                      to="/coffee/"
                    >
                      Our coffee
                    </Link>
                  </li>
                  <li
                    className="footer__item"
                  >
                    <Link
                      replace={false}
                      to="/goods/"
                    >
                      For your pleasure
                    </Link>
                  </li>
                </ul>
              </div>,
              "className": "row",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <ul
                  className="footer"
                >
                  <li
                    className="footer__item"
                  >
                    <Link
                      replace={false}
                      to="/"
                    >
                      <img
                        alt="logo"
                        src="Logo_black.svg"
                      />
                    </Link>
                  </li>
                  <li
                    className="footer__item"
                  >
                    <Link
                      replace={false}
                      to="/coffee/"
                    >
                      Our coffee
                    </Link>
                  </li>
                  <li
                    className="footer__item"
                  >
                    <Link
                      replace={false}
                      to="/goods/"
                    >
                      For your pleasure
                    </Link>
                  </li>
                </ul>,
                "className": "col-lg-5 offset-lg-4",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <li
                      className="footer__item"
                    >
                      <Link
                        replace={false}
                        to="/"
                      >
                        <img
                          alt="logo"
                          src="Logo_black.svg"
                        />
                      </Link>
                    </li>,
                    <li
                      className="footer__item"
                    >
                      <Link
                        replace={false}
                        to="/coffee/"
                      >
                        Our coffee
                      </Link>
                    </li>,
                    <li
                      className="footer__item"
                    >
                      <Link
                        replace={false}
                        to="/goods/"
                      >
                        For your pleasure
                      </Link>
                    </li>,
                  ],
                  "className": "footer",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <Link
                        replace={false}
                        to="/"
                      >
                        <img
                          alt="logo"
                          src="Logo_black.svg"
                        />
                      </Link>,
                      "className": "footer__item",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "children": <img
                          alt="logo"
                          src="Logo_black.svg"
                        />,
                        "replace": false,
                        "to": "/",
                      },
                      "ref": null,
                      "rendered": Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "alt": "logo",
                          "src": "Logo_black.svg",
                        },
                        "ref": null,
                        "rendered": null,
                        "type": "img",
                      },
                      "type": [Function],
                    },
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <Link
                        replace={false}
                        to="/coffee/"
                      >
                        Our coffee
                      </Link>,
                      "className": "footer__item",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "children": "Our coffee",
                        "replace": false,
                        "to": "/coffee/",
                      },
                      "ref": null,
                      "rendered": "Our coffee",
                      "type": [Function],
                    },
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <Link
                        replace={false}
                        to="/goods/"
                      >
                        For your pleasure
                      </Link>,
                      "className": "footer__item",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "children": "For your pleasure",
                        "replace": false,
                        "to": "/goods/",
                      },
                      "ref": null,
                      "rendered": "For your pleasure",
                      "type": [Function],
                    },
                    "type": "li",
                  },
                ],
                "type": "ul",
              },
              "type": "div",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Beans logo",
              "className": "beanslogo",
              "src": "Beans_logo_dark.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
